CONTRIBUTING.md

```diff
@@ -47,21 +47,19 @@ Pull requests (PRs)
 - Your PR must include [test coverage][run-tests].
 - Avoid cosmetic changes to unrelated files in the same commit.
 - Use a [feature branch][git-feature-branch] instead of the master branch.
-- Use a **rebase workflow** for small PRs.
-  - After addressing review comments, it's fine to rebase and force-push.
-- Use a **merge workflow** for big, high-risk PRs.
+- Use a _rebase workflow_ for small PRs.
+  - After addressing review comments, it's fine to force-push.
+- Use a _merge workflow_ (as opposed to "rebase") for big, high-risk PRs.
   - Merge `master` into your PR when there are conflicts or when master
     introduces breaking changes.
```

Comment:
Are we sure we want to encourage people merging `master` into a PR, _especially_ a big one that is likely to get merged as opposed to squashed?

Since we now have a separate section on "Merging to master", we could remove these lines (or, better, replace them with a recommendation for contributors to "rebase on master instead of merging master into PR").

Reply:
can revisit this later, but rebase causes different issues if there are lots of review comments already and many conflicting commits.